rust/counter/dfx.json

```diff
@@ -0,0 +1,23 @@
+{
+  "canisters": {
+    "counter": {
+      "candid": "counter.did",
+      "package": "counter",
+      "type": "rust"
+    }
+  },
+  "defaults": {
+    "build": {
+      "args": "",
+      "packtool": ""
+    }
+  },
+  "dfx": "0.11.1",
```

Comment:
```suggestion
```
Can we remove this line, please? With it removed, it will always run the currently installed version; with the line, as soon as a new version is released, this will cause issues.